### PR TITLE
Add security key

### PR DESCRIPTION
**Pull request by neetuy**

### CODE REVIEW

File: app/components/Bible/styles.js [hunk 1]
```diff
@@ -1,245 +1,244 @@
...
+            justifyContent: 'center'
+        },
+
+        bottomBarAudioCenter: {
+            borderRadius: 32,
+            margin: 8,
+            position: 'absolute',
+            bottom: 20,
+            left: 0,
+            // width: 56,
```
Comment (faithfulmaster) [inline app/components/Bible/styles.js:155]: remove commented lines

File: app/components/FlowLayout.js [hunk 2]
```diff
@@ -31,71 +27,71 @@ import {connect} from 'react-redux'
...
 class FlowLayout extends Component {
-	
+
 	constructor(props) {
 		super(props);
-    }
+	}
 
 	render() {
 		// console.log('FLOW LAYOUT BCV REF ',this.props.bcvRef)
```
Comment (faithfulmaster) [inline app/components/FlowLayout.js:43]: remove console.log

File: app/components/FlowLayout.js [hunk 3]
```diff
@@ -31,71 +27,71 @@ import {connect} from 'react-redux'
...
+					<TouchableOpacity onPress={() => {
+						this.props.openReference(value, position)
 					}}>
-					<View style={[styles.corner,{backgroundColor:'transparent'}]}>
-						<Text style={this.props.styles.textStyle}>{this.props.dataValue.bookName} {this.props.dataValue.chapterNumber}:{value}</Text>
-                       {/* {
+						<View style={[styles.corner, { backgroundColor: 'transparent' }]}>
+							<Text style={this.props.styles.textStyle}>{this.props.dataValue.bookName} {this.props.dataValue.chapterNumber}:{value}</Text>
+							{/* {
 						   this.props.dataValue.verses.length === 1 ? null :  <Icon name="clear" style={this.props.styles.iconReferClose}
```
Comment (faithfulmaster) [inline app/components/FlowLayout.js:53]: remove commented code

File: app/screens/Search/index.js [hunk 4]
```diff
@@ -304,10 +327,12 @@ class Search extends Component {
       tabsData: [], searchedResult: [],
       sourceId: item.sourceId, languageCode: item.languageCode,
       languageName: item.languageName, versionCode: item.versionCode,
-      downloaded: item.downloaded, books: item.books
+      downloaded: item.downloaded, books: item.books,metadata:item.metadata
+
     })
   }
   render() {
+    console.log(" STATE Metadata ",this.state.metadata && this.state.metadata[0].revision)
```
Comment (faithfulmaster) [inline app/screens/Search/index.js:335]: Remove console.log

Comment (neetuy) [reply app/screens/Search/index.js]: removed console from file and pushed code to same branch.

**Pull request merged**